configuration/configurator/schemas/SchemaAttributes.cpp

```diff
@@ -195,15 +195,17 @@ void CAttribute::populateEnvXPath(StringBuffer strXPath, unsigned int index)
 {
     assert(this->getName() != nullptr);
 
-    const char *pChar = strrchr(strXPath.str(),'[');
-    assert(pChar != nullptr && strlen(pChar) >= 3);
+
```

Comment:
Not immediately obvious why it would core. But given the variable is not used (more commented out code) not sure why it was even there.

This code is a bit of a mess.